CHANGELOG.md

```diff
@@ -985,7 +966,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Events in Web Component indicating whether Mission Zero criteria have been met (#113)
 
-[unreleased]: https://github.com/RaspberryPiFoundation/editor-ui/compare/v0.28.5...HEAD
+[unreleased]: https://github.com/RaspberryPiFoundation/editor-ui/compare/v0.28.6...HEAD
+[0.28.6]: https://github.com/RaspberryPiFoundation/editor-ui/releases/tag/v0.28.6
```

Comment:
Is it normal that this link doesn't go anywhere?

Reply:
ah, its will I've just not made the release in github yet. It'll link [here](https://github.com/RaspberryPiFoundation/editor-ui/releases/tag/untagged-84270242bb599dc24a5e) when this gets published